README.md

```diff
@@ -254,25 +257,24 @@ From your view controller:
 import SwiftyDropbox
 
 func myButtonInControllerPressed() {
-
-    // Use only one of these two flows at once:
-
-    // Legacy authorization flow that grants a long-lived token.
-    DropboxClientsManager.authorizeFromController(UIApplication.shared,
-                                                  controller: self,
-                                                  openURL: { (url: URL) -> Void in
-                                                    UIApplication.shared.openURL(url)
-                                                  })
-
-  // New: OAuth 2 code flow with PKCE that grants a short-lived token with scopes.
-  let scopeRequest = ScopeRequest(scopeType: .user, scopes: ["account_info.read"], includeGrantedScopes: false)
-  DropboxClientsManager.authorizeFromControllerV2(
-      UIApplication.shared,
-      controller: self,
-      loadingStatusDelegate: nil,
-      openURL: { (url: URL) -> Void in UIApplication.shared.openURL(url) },
-      scopeRequest: scopeRequest
-  )
+    // OAuth 2 code flow with PKCE that grants a short-lived token with scopes, and performs refreshes of the token automatically.
```

Comment:
Have we already switched to short-lived tokens? If not, will it have user-facing impact?